docs/utilities/animations.md

```diff
@@ -1385,7 +1385,7 @@ export class ModalExample {
     }
 
     const leaveAnimation = (baseEl: any) => {
-      return enterAnimation(baseEl).direction('reverse');
+      return this.enterAnimation(baseEl).direction('reverse');
```

Comment:
`enterAnimation` is declared in this scope (the `presentModal` method) rather than as a component method, so this change isn't needed (and will cause an error if pasted in).